### PR TITLE
Make fieldValue optional

### LINKED ISSUE #175: [HTTP] fieldValue should not be mandatory in responses
As seen in the TM of the TDD at https://w3c.github.io/wot-discovery/#directory-api-spec, the `fieldValue` should not be mandatory in responses since the response can be a variable.

### PR DESCRIPTION
**Pull request by egekorkan**
fixes #175

***

### :boom: Error: 500 Internal Server Error :boom: ###

[PR Preview](https://github.com/tobie/pr-preview#pr-preview) failed to build. _(Last tried on Sep 28, 2022, 2:55 PM UTC)_.

<details>
<summary>More</summary>

PR Preview relies on a number of web services to run. There seems to be an issue with the following one:

:rotating_light: [Spec Generator](https://www.w3.org/2015/labs/) - Spec Generator is the web service used to build specs that rely on ReSpec.

:link: [Related URL](https://labs.w3.org/spec-generator/?type=respec&url=https%3A%2F%2Fraw.githubusercontent.com%2Fw3c%2Fwot-binding-templates%2Fd95d21c3b57ba9f83416036c87ffb16b0e6a20e6%2Findex.html%3FisPreview%3Dtrue)

```
🕵️‍♀️  That doesn't seem to be a ReSpec document. Please check manually: https://labs.w3.org/spec-generator/uploads/Xa47jM/index.html?isPreview=true%3FisPreview%3Dtrue&publishDate=2022-09-28
```

_If you don't have enough information above to solve the error by yourself (or to understand to which web service the error is related to, if any), please [file an issue](https://github.com/tobie/pr-preview/issues/new?title=Error%20not%20surfaced%20properly&body=See%20w3c/wot-binding-templates%23180.)._
</details>

### CODE REVIEW

**Comment by egekorkan** [comment]
@lu-zero have a look as well :)

**Review by relu91 — APPROVED** [review]
LGTM, however, it seems that this PR contains additional commits coming from #181.

File: bindings/protocols/http/index.html
Comment (lu-zero) [inline bindings/protocols/http/index.html:96]: `</code>` seems spurious.

**Comment by egekorkan** [comment]
> LGTM, however, it seems that this PR contains additional commits coming from #181.

Ah my bad, let's wait for that and rebase before merging :)

File: bindings/protocols/http/index.html
Comment (egekorkan) [reply bindings/protocols/http/index.html]: oversight from my side, done :)

**Comment by egekorkan** [comment]
Call of 28.09:
- after merging 181, @egekorkan will merge this one after a rebase

**Pull request merged**
